api/src/routes/edit.ts

```diff
@@ -319,6 +341,72 @@ edit.put("/object/:pid/state", requireToken, pidSanitizer, bodyParser.text(), as
 });
 
 const pidAndParentPidSanitizer = sanitizeParameters({ pid: pidSanitizeRegEx, parentPid: pidSanitizeRegEx });
+edit.put(
+    "/object/:pid/parent/:parentPid",
+    requireToken,
+    pidAndParentPidSanitizer,
+    bodyParser.text(),
+    async function (req, res) {
+        try {
+            const pid = req.params.pid;
```

Comment:
You can do.` const { pid, parentPid: parent} = req.params;`

Reply:
Hooray for destructuring! I've optimized things a bit in b8e764d653501fc8bb90239b58a568ea79d673c1.